NCSU-App-Development-Club/Marigold-Android
# 10 post list items

### Link to Ticket
- Link to the related ticket: [#10](https://github.com/NCSU-App-Development-Club/Marigold-Android/issues/10)
- 
### Description
Adds a post list item composable to the shared Home Screen file.

### Images and Screenshots
<img width="422" alt="Screenshot 2024-01-24 at 10 51 43 AM" src="https://github.com/NCSU-App-Development-Club/Marigold-Android/assets/44850083/052bad26-dc80-45ea-be44-48dffd6531c2">


### Steps to Test
1. In the home/shared/composables.kt file, inspect the PostPreview
2. Play with line count of message to observe line limit.


### Code Checking Checklist
- [X] I have followed the coding standards and guidelines.
- [X] My code includes proper comments and documentation.
- [X] I have tested my code thoroughly, including edge cases.
- [X] I have ensured that my code doesn't contain any sensitive information (e.g., passwords, keys).
- [X] My changes don't break any existing functionalities.

### Reviewers


### Additional Notes
